youtube_dl/extractor/openload.py

```diff
@@ -254,7 +254,15 @@ class OpenloadIE(InfoExtractor):
                         (?:f|embed)/
                         (?P<id>[a-zA-Z0-9-_]+)
                     ''' % _DOMAINS
+    _EMBED_WORD = 'embed'
+    _STREAM_WORD = 'f'
+    _REDIR_URL = 'stream'
```

Comment:
use an appropriate name for the variable(it's not a URL).

Reply:
Changed to `_REDIR_WORD`.